### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by raquel-ucl**
Added some stuff in the README.

### CODE REVIEW

**Comment by raquel-ucl** [comment]
Ready for review. Jens, please also have a look at the wiki which contains some documentation on SOAP stuff. 
Let me know if there's something that needs correction or tidy up, or something else you think I should add.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,141 @@
...
+ORACC is the [Open Richly Annotated Cuneiform Corpus](http://oracc.org). 
+It provides open-access, standards-based publication platforms, research tools 
+and teaching resources for Assyriology and ancient Near Eastern History, 
+hosting around 40 academic research projects worldwide. 
+
+ORACC has become established as one of the core online resources in the world 
+of ancient Near Eastern studies. It originated in an AHRC-funded research project 
+[Prof. Eleanor Robson](https://www.ucl.ac.uk/history/people/academic-staff/eleanor-robson) 
+ran at the University of Cambridge several years ago and is now continuing to 
+run from University College London.
```
Comment (jenshnielsen) [inline README.md]: and University of Pennsylvania?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,141 @@
...
+##What is Nammu?
+Nammu is a desktop GUI that enables ORACC content creators to edit, check and 
+upload transliterations of Sumerian texts in the form of text files in 
+[ASCII Transliteration Format](http://oracc.museum.upenn.edu/doc/help/editinginatf/) 
+(ATF). It is being developed as a text editor with extended functionality.
+
+At the time of writing, ORACC content creators use an Emacs plugin for edition,
+validation and lemmatisation of ATF files. This plugin can only be installed as 
+part of [Emacs](http://oracc.museum.upenn.edu/doc/help/usingemacs/emacssetup/index.html), 
+which has a steep learning curve for some users.
```
Comment (jenshnielsen) [inline README.md]: delete "for some users"?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,141 @@
...
+[ASCII Transliteration Format](http://oracc.museum.upenn.edu/doc/help/editinginatf/) 
+(ATF). It is being developed as a text editor with extended functionality.
+
+At the time of writing, ORACC content creators use an Emacs plugin for edition,
+validation and lemmatisation of ATF files. This plugin can only be installed as 
+part of [Emacs](http://oracc.museum.upenn.edu/doc/help/usingemacs/emacssetup/index.html), 
+which has a steep learning curve for some users. 
+
+With Nammu, we intend to make a user friendly tool that would replace the use
+of the ORACC Emacs plugin. This would help lower the access barriers to the use
```
Comment (jenshnielsen) [inline README.md]: will help?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,141 @@
...
+of ORACC, enabling more projects to adopt it.
+
+Nammu is currently being developed by the [UCL Research Software Development Group](https://www.ucl.ac.uk/research-it-services/about/research-software-development).
+
+
+## Validation of ATF files
+
+In order to validate ATF files, Nammu uses two approaches:
+* Online validation against ORACC server.
+* Offline validation using pyORACC, the new ATF parser developed by UCL RSDT.
```
Comment (jenshnielsen) [inline README.md]: Link to pyoracc repository?

**Comment by jenshnielsen** [comment]
Why not merged?

**Comment by raquel-ucl** [comment]
Merged now :)

**Pull request merged**